cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -3576,31 +3572,31 @@
 ;; Helper for creating `divss` instructions.
 (decl x64_divss (Xmm XmmMem) Xmm)
 (rule (x64_divss src1 src2)
-      (xmm_rm_r_unaligned (SseOpcode.Divss) src1 src2))
+      (x64_divss_a src1 src2))
 (rule 1 (x64_divss src1 src2)
       (if-let true (use_avx))
       (xmm_rmir_vex (AvxOpcode.Vdivss) src1 src2))
 
 ;; Helper for creating `divsd` instructions.
 (decl x64_divsd (Xmm XmmMem) Xmm)
 (rule (x64_divsd src1 src2)
-      (xmm_rm_r_unaligned (SseOpcode.Divsd) src1 src2))
+      (x64_divsd_a src1 src2))
```

Comment:
A nit to add the rule priority and, in these very short cases, fit it on a single line:

```
(rule 0 (x64_divsd src1 src2) x64_divsd_a src1 src2))
```